aif360/sklearn/postprocessing/__init__.py

```diff
@@ -175,7 +175,7 @@ def predict_proba(self, X):
         y_score = pd.DataFrame(y_score, index=X.index).squeeze('columns')
         return self.postprocessor_.predict_proba(y_score)
 
-    @if_delegate_has_method('postprocessor_')
+    @available_if('postprocessor_')
```

Comment:
How do we test if this is working? Have we ran the code end to end to check if this is working?